### PR TITLE
Move Pr 350 from old docs

### PR DESCRIPTION
**Pull request by ipopescu**
Moving content from the old docs here: https://github.com/casper-network/docs/pull/350/files.
@Buddhika-Pathirathna please review next sprint when you get a chance. Thank you!

### CODE REVIEW

**Comment by ipopescu** [comment]
@Buddhika-Pathirathna when you get a chance, please review this small PR. Thank you :).

File: source/docs/casper/operators/hardware.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
...
 -   4 Cores
 -   32 GB Ram
--   1 TB disk (Both SSDs and HDDs work, but SSDs perform initial sync much faster.)
+-   1 TB SSD or network SSD backed disk
+-   Linux machine running Ubuntu 18.04 or 20.04 (preferred) with a Rust build environment.
 
-Linux machine running Ubuntu 18.04 or 20.04 with a Rust build environment. Reference <https://github.com/casper-network/casper-node> for details.
+**Notes**:
+
+SDD is required because HDD cannot perform random writes at the performance needed to keep in sync with the full speed of the network.
```
Comment (Buddhika-Pathirathna) [inline source/docs/casper/operators/hardware.md:14]: Does this should be 'SSD '?

Comment (ipopescu) [reply source/docs/casper/operators/hardware.md:14]: Yes @Buddhika-Pathirathna , it's a solid-state drive.

**Review by Buddhika-Pathirathna — APPROVED** [review]

File: source/docs/casper/operators/hardware.md [hunk 2]
```diff
@@ -2,10 +2,17 @@
...
+
+For non-archival nodes, disc usage will drop once data recovery is implemented. It is safe to slowly increase disc space as needed if monitored and on a server capable of this.
```
[the remark this answers is not in the document]
Comment (ipopescu) [reply source/docs/casper/operators/hardware.md]: Thanks @Buddhika-Pathirathna ! I've implemented your suggestion. It is clearer.

**Pull request merged**